s4e-backend/src/main/java/pl/cyfronet/s4e/controller/SceneController.java

```diff
@@ -0,0 +1,57 @@
+package pl.cyfronet.s4e.controller;
+
+import io.swagger.annotations.ApiOperation;
+import io.swagger.annotations.ApiResponse;
+import io.swagger.annotations.ApiResponses;
+import lombok.RequiredArgsConstructor;
+import org.springframework.format.annotation.DateTimeFormat;
+import org.springframework.web.bind.annotation.*;
+import pl.cyfronet.s4e.controller.response.SceneResponse;
+import pl.cyfronet.s4e.service.SceneService;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.LocalTime;
+import java.time.YearMonth;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import static pl.cyfronet.s4e.Constants.API_PREFIX_V1;
+
+@RestController
+@RequestMapping(API_PREFIX_V1)
+@RequiredArgsConstructor
+public class SceneController {
+    private final SceneService sceneService;
+
+    @ApiOperation(value = "View a list of scenes")
+    @ApiResponses({
+            @ApiResponse(code = 200, message = "Successfully retrieved list")
+    })
+    @GetMapping("/scenes/productId/{productId}")
```

Comment:
not the scope of this PR but, shouldnt this endpoint look like: products/{productId}/scenes :)